brave/brave-core
# Cleanup ad server/catalog project structure

Resolves https://github.com/brave/brave-browser/issues/23020

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm catalog is downloaded after fresh install
- Confirm catalog is downloaded again after delay defined in catalog under "ping" node (in milliseconds)
